packages/terra-search-field/src/SearchField.jsx

```diff
@@ -26,6 +26,11 @@ const propTypes = {
   searchDelay: PropTypes.number,
 
   /**
+   * Function to trigger when user changes the input value. Provide a function to create a controlled input.
+   */
+  onChange: PropTypes.func,
```

Comment:
onChange needs to be able to be used in concert with a value prop in order for this to be a controlled component.  As is, you'll be telling the consumer what changed, but they won't be able to update the input value.

Reply:
added a value prop for the consumer to set